src/wasm/wasm-type.cpp

```diff
@@ -159,7 +167,7 @@ struct SubTyper {
   // otherwise.
   std::unordered_set<std::pair<HeapType, HeapType>> seen;
   bool isSubType(Type a, Type b);
-  bool isSubType(HeapType a, HeapType b);
+  std::optional<size_t> isSubType(HeapType a, HeapType b);
```

Comment:
It's not obvious what this returns now?

Reply:
The distance in the subtype hierarchy between `a` and `b` iff `a <: b`. I'll add a comment.